draft-rha-jose-hpke-encrypt.md

```diff
@@ -208,51 +219,47 @@ This example demonstrates the representaton of an encapsulated key as a JWK.
 
 ### Integrated Encryption
 
-In Integrated Encryption mode, HPKE is employed to directly encrypt the (compressed) plaintext, and the resulting ciphertext is included in the JWE ciphertext. 
+This mode only supports a single recipient and JWE Compact serialization.
 
-In this mode, JWE Compact serialization MUST be used. The sender MUST specify the 'epk' and 'alg' parameters in the protected header to indicate the use of HPKE. Optionally, the protected header MAY contain the 'kid' parameter used to identify the static recipient public key used by the sender. 
+HPKE is employed to directly encrypt the plaintext, and the resulting ciphertext is included in the JWE ciphertext. 
 
-The Single-Shot APIs specified in Section 6 of {{RFC9180}} for encryption and decryption cannot be used. This is because they require an 'aad' parameter, which takes the Encoded Protected Header comprising of 'ek' as input.
```

Comment:
This sentence did not make sense to me... I don't know if the single shot APIs matter at all to this spec, but we do use `aad` in this spec.

Reply:
As with the COSE-HPKE document, we shouldn't focus too much on the APIs

Reply:
Yes.